README.md

```diff
@@ -21,19 +21,33 @@ You can install them using yum, but beware some of them are on epel and centos-r
 # yum install centos-release-scl
 # yum install llvm-toolset-7
 ```
+
+You need to create a postgres user to execute PostgreSQL.
+
+```
+# useradd postgres
+# passwd postgres
```

Comment:
Is the password necessary?
I'm not a fan of exposing passwords even if they are samples.